src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -11794,17 +11798,19 @@ pbs_python_set_os_environ(char *env_var, char *env_val)
 		pbs_python_write_error_to_log(log_buffer);
 		Py_CLEAR(os_mod_obj);
 		return (-1);
+
 	}
 
 	if ((os_env_dict =
-		PyObject_GetAttrString(os_mod_env, "data")) == NULL) {
+		PyObject_GetAttrString(os_mod_env, "_data")) == NULL) {
```

Comment:
Just confirming: is this change intentional?

Reply:
Yes. os.environ.data was renamed in Python 3.2 to os.environ._data
[nehap@neha20 pbspro]$ python3.6
Python 3.6.8 (default, Apr 25 2019, 21:02:35) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> dir(os.environ)
['_MutableMapping__marker', '__abstractmethods__', '__class__', '__contains__', '__delattr__', '__delitem__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__reversed__', '__setattr__', '__setitem__', '__sizeof__', '__slots__', '__str__', '__subclasshook__', '__weakref__', '_abc_cache', '_abc_negative_cache', '_abc_negative_cache_version', '_abc_registry', '_data', 'clear', 'copy', 'decodekey', 'decodevalue', 'encodekey', 'encodevalue', 'get', 'items', 'keys', 'pop', 'popitem', 'putenv', 'setdefault', 'unsetenv', 'update', 'values']
>>> 
[nehap@neha20 pbspro]$ python2.7
Python 2.7.5 (default, Jun 20 2019, 20:27:34) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> dir(os.environ)
['__cmp__', '__contains__', '__delitem__', '__doc__', '__getitem__', '__hash__', '__init__', '__iter__', '__len__', '__module__', '__repr__', '__setitem__', 'clear', 'copy', 'data', 'fromkeys', 'get', 'has_key', 'items', 'iteritems', 'iterkeys', 'itervalues', 'keys', 'pop', 'popitem', 'setdefault', 'update', 'values']
>>>